packages/ember-principled-forms/addon/components/principled-form-field/component.ts

```diff
@@ -2,6 +2,8 @@ import Component from '@ember/component';
 import { assert } from '@ember/debug';
 import { action } from '@ember-decorators/object';
 
+// DISCUSS: Do we really need a lodash dependency?
```

Comment:
Probably not. Lodash is small enough that I don't have strong feelings either way here, but it's also easy enough *for the things we're doing* to just do it ourselves.